tests/xfstests/generate_report.pm

```diff
@@ -15,34 +15,51 @@ use strict;
 use 5.018;
 use warnings;
 use base "opensusebasetest";
+use serial_terminal "select_virtio_console";
+use File::Basename;
 use testapi;
 use ctcs2_to_junit;
 
-my $LOG_FILE   = "/tmp/xfstests.log";
+my $STATUS_LOG = "/tmp/status.log";
+my $LOG_DIR    = "/tmp/log";
+my $KDUMP_DIR  = "/tmp/kdump";
 my $JUNIT_FILE = "/tmp/output.xml";
 
+
```

Comment:
There are many duplicated empty lines in your changes, check your changed files to keep one empty line.